src/include/daos/common.h

```diff
@@ -646,6 +646,7 @@ bool daos_hhash_link_delete(struct d_hlink *hlink);
 
 /* NVMe shared constants */
 #define DAOS_NVME_SHMID_NONE	-1
+#define DAOS_NVME_MEM_SIZE_DEFAULT 0
```

Comment:
Minor, I think we can use 0 directly in code instead of this macro, or rename it as DAOS_NVME_MEM_PRIMARY?

Reply:
Ok, I have changed it because I made a minor update.